src/ORM/RulesChecker.php

```diff
@@ -47,6 +47,9 @@ class RulesChecker extends BaseRulesChecker
      * ### Options
      *
      * - `allowMultipleNulls` Allows any field to have multiple null values. Defaults to false.
+     * - 'message' sets a custom error message.
+     *
+     * Other options will be applied to the query in `\Cake\ORM\Table::exists()`
```

Comment:
I don't like this mixing of method specific options with options for query clauses.

Reply:
Shall I introduce a new third argument (`$queryArgs`)?

Reply:
If we add a $finder arg to exists() then an additional `finder` in options would suffice.